Makefile

```diff
@@ -235,7 +235,7 @@ $(build_dir)/%.d : $(src_dir)/%.[c,S]
 
 $(objs-y):
 	@echo "Compiling source	$(patsubst $(cur_dir)/%, %, $<)"
-	@$(cc) $(CFLAGS) -c $< -o $@
+	-@$(cc) $(CFLAGS) -c $< -o $@
```

Comment:
We should not be suppressing build errors.

Reply:
Should be fixed in the recent commit.